src/editor/components/graph/node-types.ts

```diff
@@ -0,0 +1,76 @@
+import { Node as ReactFlowNode } from 'react-flow-renderer/dist/nocss/types/nodes'
+import { XYPosition } from 'react-flow-renderer/dist/nocss/types/utils'
+
+class Node implements ReactFlowNode {
+  id: string
+  dragHandle: string
+  type: string
+  data: any
+  position: XYPosition
+
+  constructor( type: string, data: any ) {
+    this.id = 'id-' + Date.now()
```

Comment:
Good for now but we might want to use something else here? Do we still have the uuid package?

Reply:
good idea